integration/deploy/deploy_remote_test.go

```diff
@@ -0,0 +1,77 @@
+//go:build integration
+// +build integration
+
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deploy
+
+import (
+	"testing"
+
+	"github.com/okteto/okteto/integration"
+	"github.com/okteto/okteto/integration/commands"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	parentManifestContent = `
+deploy:
+  remote: true
+  commands:
+    - okteto deploy -f other-okteto.yml --remote`
+
+	childManifestContent = `
+deploy:
+  image: aquasec/trivy:latest
+  commands:
+    - trivy -q image alpine:3.14`
+)
+
+// TestDeployInDeployRemote test the scenario where an okteto deploy is run inside an okteto deploy in remote
+// image base for the child deploy should be the specified at the child manifest and not the parent
+func TestDeployInDeployRemote(t *testing.T) {
```

Comment:
when I execute this test locally, it fails with `sh: 1: trivy: not found`.

Does it pass for you, @teresaromero?

Reply:
locally are you using the custom CLI image? if not it will fail becaue the changes are on the remote runner

Reply:
no I wasn't, thanks!